types/node/index.d.ts

```diff
@@ -50,6 +50,16 @@
 // NOTE: These definitions support NodeJS and TypeScript 3.7.
 // Typically type modifications should be made in base.d.ts instead of here
 
+declare namespace NodeJS {
+    interface Dict<T> {
+        [key: string]: T;
+    }
```

Comment:
My plan is to make this type specific to 4.2+, but to begin with I just wanted to test the versioning mechanism works.

When I try to run `npm run test-all` I get this error:

```
> definitely-typed@0.0.3 test-all /Users/oliverash/Development/DefinitelyTyped
> node node_modules/@definitelytyped/dtslint-runner/dist/index.js --path .

Using local DefinitelyTyped at .
Parsing definitions...
Found 7500 packages.
Parsing in 16 child process(es)...
Error: Unused file ./types/node/ts3.4/assert.d.ts (used files: ["index.d.ts","node-tests.ts","../test/assert.ts","../test/async_hooks.ts","../test/child_process.ts","../test/cluster.ts","../test/constants.ts","../test/crypto.ts","../test/dgram.ts","../test/dns.ts","../test/events.ts","../test/fs.ts","../test/global.ts","../test/http.ts","../test/http2.ts","../test/module.ts","../test/net.ts","../test/os.ts","../test/path.ts","../test/perf_hooks.ts","../test/process.ts","../test/readline.ts","../test/repl.ts","../test/stream.ts","../test/string_decoder.ts","../test/tls.ts","../test/tty.ts","../test/util.ts","../test/v8.ts","../test/vm.ts","../test/worker_threads.ts","../test/zlib.ts","tsconfig.json","tslint.json"])
```

Does anyone know why this is happening?

Reply:
Nevermind, I fixed it by moving the `declare` after the `reference`s. 🤷‍♂️